src/createServer.js

```diff
@@ -1,8 +1,46 @@
 'use strict';
 
+const http = require('http');
+const path = require('path');
+const fs = require('fs');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const pathname = req.url;
+    let normalizedPath = pathname.replace('/file', '');
+
+    if (normalizedPath.length > 0) {
+      normalizedPath = path.join(__dirname, '..', 'public', normalizedPath);
+    } else {
+      normalizedPath = 'public/index.html';
+    }
+
+    res.setHeader('content-type', 'text/plain');
```

Comment:
The content type is set to 'text/plain', which may not be suitable for all files served. Consider setting the content type dynamically based on the file extension to ensure correct rendering in the browser.